src/components/Ads/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import { View, Text, StyleSheet, Platform } from 'react-native'
+import { AdMobBanner } from 'react-native-admob'
+
```

Comment:
Since there's no need for this to be a class component, I'd probably use a functional component here.

Reply:
I agree, a functional component here would be perfect, along with a state handler for the `adID` so we don't have to recalculate every re-render. If you need help feel free to reach out!